src/pages/training.js

```diff
@@ -1,15 +1,324 @@
 import React from 'react';
```

Comment:
The image "static/images/clients/twilio.png" shouldn't have the "and more" in the same image. That should be as text.